selfdrive/car/toyota/values.py

```diff
@@ -298,23 +298,34 @@ class CAR:
 
 FW_VERSIONS = {
   CAR.AVALON: {
-    (Ecu.esp, 0x7b0, None): [b'F152607060\x00\x00\x00\x00\x00\x00'],
+    (Ecu.esp, 0x7b0, None): [
+      b'F152607110\x00\x00\x00\x00\x00\x00',
+      b'F152607180\x00\x00\x00\x00\x00\x00',
+    ],
     (Ecu.dsu, 0x791, None): [
       b'881510705200\x00\x00\x00\x00',
       b'881510701300\x00\x00\x00\x00',
+      b'881510703200\x00\x00\x00\x00',
+    ],
+    (Ecu.eps, 0x7a1, None): [
+      b'8965B41090\x00\x00\x00\x00\x00\x00',
     ],
-    (Ecu.eps, 0x7a1, None): [b'8965B41051\x00\x00\x00\x00\x00\x00'],
     (Ecu.engine, 0x7e0, None): [
       b'\x0230721100\x00\x00\x00\x00\x00\x00\x00\x00A0C01000\x00\x00\x00\x00\x00\x00\x00\x00',
       b'\x0230721200\x00\x00\x00\x00\x00\x00\x00\x00A0C01000\x00\x00\x00\x00\x00\x00\x00\x00',
     ],
+    (Ecu.engine, 0x700, None): [
+      b'\x01896630731000\x00\x00\x00\x00',
```

Comment:
this is wrong, should be
```suggestion
      b'\x01896630738000\x00\x00\x00\x00',
```